Implementation/syntax highlight

cicerotcv opened the pull request:
Fix #24

toshikurauchi commented on frontend/frontend-dev/src/components/CodeVisualizer/index.tsx:
```diff
@@ -21,7 +21,7 @@ export function CodeVisualizer({
         lineNumberStyle={{
           fontWeight: "bold",
-          color: "#c0c0c050",
+          color: "##555a7b",
```
I think there's an extra `#` here

toshikurauchi commented on frontend/frontend-dev/src/fragments/ExerciseResultVisualization/InstructorCodeExerciseResult/index.tsx:
```diff
@@ -1,9 +1,10 @@
-import { useState } from "react";
+import { Fragment, useState } from "react";
```
Do we need to import `Fragment` here?

toshikurauchi commented on frontend/frontend-dev/src/fragments/ExerciseResultVisualization/InstructorCodeExerciseResult/index.tsx:
```diff
@@ -48,11 +72,32 @@ export function CodeExerciseResult({ codeAnswers }: ICodeResultProps) {
+                    />
+                  </label>
+                ),
+                // compare: (
```
Do we need this comment?

toshikurauchi reviewed (changes requested):
Just a couple of comments, but everything looks good.

In the storybook setting values to `leftTitle`, `rightTitle` and `title` didn't do anything, is that expected?

cicerotcv commented:
> Just a couple of comments, but everything looks good.
> 
> In the storybook setting values to `leftTitle`, `rightTitle` and `title` didn't do anything, is that expected?

That's right. It's not implemented yet. It will work when the header of the CodeBlock component. I'll fix this PR addressing the comments you've made and removing these props for now.

toshikurauchi commented:
> > Just a couple of comments, but everything looks good.
> > In the storybook setting values to `leftTitle`, `rightTitle` and `title` didn't do anything, is that expected?
> 
> That's right. It's not implemented yet. It will work when the header of the CodeBlock component. I'll fix this PR addressing the comments you've made and removing these props for now.

You can keep the props, it's fine. I just wanted to confirm that they were not being used yet. Let me know when you want me to re-review your PR.
